### PR TITLE
dppl check GPU and CPU devices

### PR DESCRIPTION
**Pull request by PokhodenkoSA**

### CODE REVIEW

File: numba/dppl_config.py [hunk 1]
```diff
@@ -1,8 +1,5 @@
-dppl_present = False
-
 try:
     import dpctl
+    dppl_present = dpctl.has_gpu_queues() and dpctl.has_cpu_queues()
```
Comment (AlexanderKalistratov) [inline numba/dppl_config.py:3]: OR

Comment (diptorupd) [reply numba/dppl_config.py:3]: Argh! Yes. You are right.

Comment (PokhodenkoSA) [reply numba/dppl_config.py:3]: I doubt numba.dppl works with CPU devices.
@reazulhoque Can it work when only CPU devices available?

Comment (diptorupd) [reply numba/dppl_config.py:3]: The intent of the change is to make sure there is at least some OpenCL device (CPU or GPU) for Numba.dppl to use. We can also use dpctl.has_sycl_platforms() for this check.

Comment (reazulhoque) [reply numba/dppl_config.py:3]: @PokhodenkoSA it should work with any sycl device.

**Pull request merged**